tljh-plasma/tljh_plasma/entrypoint/entrypoint.sh

```diff
@@ -43,4 +43,4 @@ export JUPYTER_PATH=${IMAGE_DIR}/.local/share/jupyter
 cd ${IMAGE_DIR}
 
 # execute the notebook process as the given user
-exec su - $NB_USER -m -c '"$0" "$@"' -- "$@"
+exec su $NB_USER -m -c '"$0" "$@"' -- "$@"
```

Comment:
OK this seems to be doing the trick for being able able to launch the server.

If I understand correctly, specifying `-` would be equivalent to starting the process with `--login`, which conflicts with `-m` / `--preserve-environment`.

As hinted by the the error logs:

```
su: ignoring --preserve-environment, it's mutually exclusive with --login
--root-prefix: 1 required TEXT missing
Run with --help for more information.
/srv/conda/envs/notebook/bin/jupyterhub-singleuser: 7: /etc/profile.d/activate-conda.sh: [[: not found
/srv/conda/envs/notebook/bin/jupyterhub-singleuser: 20: /etc/profile.d/activate-conda.sh: mamba: not found
JUPYTERHUB_API_TOKEN env is required to run jupyterhub-singleuser. Did you launch it manually?
```

Reply:
I guess this is related to recent update of the mamba activation scripts in `repo2docker`. Removing the `-` seems to be working fine locally so we should be able to do without now:

![image](https://github.com/plasmabio/plasma/assets/591645/598960c8-bde8-4bf7-812f-6b6fd401fde6)

Reply:
Need to do a bit more testing on a test VM, but this change might break compatibility with existing environments built with an older version of `repo2docker`.